megamek/src/megamek/common/VTOL.java

```diff
@@ -273,6 +273,20 @@ public void clearBombChoices() {
         Arrays.fill(bombChoices, 0);
     }
 
+    @Override
+    public int getBombLoad(int t) {
+        // Per TacOps errata v3.0, movement reduction is per bomb rather than per 5 bomb points
+        for (Mounted m : getBombs()) {
+            if (m.getUsableShotsLeft() > 0) {
+                t--;
+            }
+        }
+        if (t < 0) {
+            t = 0;
+        }
+        return t;
+    }
```

Comment:
Well I havent tested it but this could probably be a one-liner  :D
return Math.max(0, t - getBombs().stream().filter(m-> m-getUsableShotsLeft() > 0).count());
(really unimportant, it just crossed my mind)